doc/vhpidirect/examples/quickstart.rst

```diff
@@ -157,3 +157,17 @@ data through an access/pointer. This approach is more verbose than others, but i
 2008 without modification and without requiring :option:`-frelaxed`. Moreover, it enhances encapsulation, as it provides
 a user-defined API between VHDL and C, which can improve maintainability when sources are reused. As a matter of fact,
 this approach is found in verification projects such as `VUnit <http://vunit.github.io/>`_ and `OSVVM <https://osvvm.org/>`_.
+
+.. _COSIM:VHPIDIRECT:Examples:quickstart:shrecord:
+
+:cosimtree:`shrecord <vhpidirect/quickstart/sharedvar/shrecord>`
```

Comment:
For consistency, please create a header for 'shint' too. No need to modify anything. Just add a header before the previous paragraph.